Apply sentence case capitalization

GPhMorin opened the pull request:
Note: I might have forgotten some strings or mistakenly capitalized others. Particularly, I didn't touch the Help text strings, which seems to capitalize buttons (e.g. "Click on the 'Close Account' button" should become "Click on the 'Close account' button")

Also I didn't know if I had to add periods at some places. Some places such as "webfinger" seems to be inconsistent with punctuation.

GPhMorin committed:
Apply sentence case capitalization

Note: I might have forgotten some strings or mistakenly capitalized others. Particularly, I didn't touch the Help text strings, which seems to capitalize buttons (e.g. "Click on the 'Close Account' button" should become "Click on the 'Close account' button")

Also I didn't know if I had to add periods at some places. Some places such as "webfinger" seems to be inconsistent with punctuation.

Flaburgan commented on config/locales/diaspora/en.yml:
```diff
@@ -22,7 +22,7 @@ en:
   ignore: "Ignore"
   undo: "Undo?"
-  or: "or"
+  or: "Or"
```
I guess this is included inside a sentence, so it should not be capitalized.

maliktunga replied:
Done.

Flaburgan commented on config/locales/diaspora/en.yml:
```diff
@@ -928,7 +928,7 @@ en:
     new_profile_photo:
       upload: "Upload a new profile photo!"
-      or_select_one_existing: "or select one from your already existing %{photos}"
+      or_select_one_existing: "Or select one from your already existing %{photos}"
```
Same remark here, this is the end of the previous sentence so no O on "or" imo.

maliktunga replied:
Done.

Flaburgan commented on config/locales/diaspora/en.yml:
```diff
@@ -1004,11 +1004,11 @@ en:
+      email: "Email"
+      username: "Username"
+      password: "Password"
+      password_confirmation: "Password confirmation"
```
We have to keep capitals here if you want to have this style because this text is included in placeholders which can't be capitalized with CSS.

maliktunga replied:
Done.

Flaburgan commented on config/locales/diaspora/en.yml:
```diff
@@ -1273,31 +1273,31 @@ en:
       stream_preferences: "Stream preferences"
-      show_community_spotlight: "Show Community Spotlight in stream"
-      show_getting_started: 'Show Getting Started hints'
+      show_community_spotlight: "Show community spotlight in stream"
```
I'm also wondering if "Community Spotlight" is a proper name and then should be capitalized.

maliktunga replied:
I'm wondering the same. Does diaspora* have kind of "branded features" like Twitter's Stream, Tweets, ReTweets, Trends, Hashtags, etc.? Twitter considers their prime features as proper nouns and capitalizes them. I think Google's Android style guide also suggests the same.

GPhMorin committed:
"or" stays lowercase

As suggested by https://github.com/diaspora/diaspora/pull/5579#discussion_r23297945

GPhMorin committed:
"or select [...]" stays lowercase

As suggested by https://github.com/diaspora/diaspora/pull/5579#discussion-diff-23298194

GPhMorin committed:
Keep capitals in placeholders

As suggested by https://github.com/diaspora/diaspora/pull/5579#discussion_r23298341

GPhMorin committed:
"Live chat" becomes "live chat"

GPhMorin committed:
Apply sentence case capitalization in documentation

I didn't complete it though.

GPhMorin committed:
"delete button" becomes "'Delete this aspect' button"

GPhMorin committed:
"edit profile page" becomes "'Edit profile' page"

GPhMorin committed:
"NewHere" becomes "newhere"

maliktunga commented:
I'm pretty much done.

jhass commented:
CI is not failing randomly ;)

maliktunga commented:
@jhass I thought it did. I don't even know CI is. What's wrong with my branch?

GPhMorin committed:
Merge pull request #2 from diaspora/develop

Update Fork

jhass commented:
Continues Integration, it runs our testsuite after each commit. You can see it's status next to your commits on Github, here's the run of your merge commit (which should've been a rebase btw ;) ): https://travis-ci.org/diaspora/diaspora/builds/47870699

maliktunga commented:
@jhass Hmm I see, but what's the relation between my commits and the CI's failure? I did not change anything in the actual functioning of Diaspora, I just changed text strings…

jhass commented:
Some of our tests assert on some of the strings you changed in order to verify that the correct code path was taken for example. And the integration tests actually automatically drive a browser, so those rely a lot on the texts.

maliktunga commented:
@jhass So what then? What should be done to fix these issues?

Flaburgan commented:
@maliktunga you also have to replace the text in the tests to make them pass.

For example, the first test is failing because:
<pre>Expected `All aspects' to contain 'All Aspects'.</pre>

You changed A to a so the test is failing. Please read https://wiki.diasporafoundation.org/Testing_workflow to understand how to run tests and fix them.

goobertron commented:
This is why a while ago I altered as many of the cucumber tests as I could to test for a variable/path rather than a string (e.g. aspects.all_aspects rather than 'All Aspects'). Unfortunately I didn't manage to alter them all!

goobertron commented:
There are also a few Title Case strings in `config/javascript/javascript.en.yml`, e.g. line 25, 'No Results Found'.

I don't think we've yet discussed specifically how to handle names of parts of the app or features, such as 'My Aspects', '#Followed Tags', 'Community Spotlight' and 'Getting Started'.

GPhMorin committed:
Apply sentence case capitalization

maliktunga commented:
@goobertron I updated the file you mentioned.

I consider that I have done enough. This is my last contribution to the Diaspora project. Feel free to continue this commit from my work. Thank you for your interest in a freedom-focused, decentralized social network.

RichKavanagh commented:
Right then, back to 200 individual commits that pass travis ci.

Flaburgan commented:
@maliktunga I don't get your last message, that means that you will not fix the test?

maliktunga commented:
Yes. I will not fix it.

On Friday, January 23, 2015, Fla <notifications@github.com> wrote:

> @maliktunga <https://github.com/maliktunga> I don't get your last
> message, that means that you will not fix the test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/5579#issuecomment-71196130>.
>

Flaburgan commented:
@goobertron wanna do it, or I'll fix them.

goobertron commented:
I'm happy to (try to) do it, but it probably won't be this week as I have some wiki things I've promised to look at first.

goobertron commented:
@maliktunga It's a shame that you don't want to continue contributing. Is there any particular reason for that?

maliktunga commented:
Oh, many reasons. Notably disinterest and lack of time. I support that you
are working in a freedom software project, decentralization and freedom of
speech are very important, but I'm not interested in Diaspora itself. I
would want it to be completely different from what it is right now, and
Loomio democracy does not share the same vision I have (first time that I
don't support democracy). Furthermore, it turns out I am not really
interested in social networks altogether. I prefer chatting with friends
and sharing emails, which Diaspora does not provide me (and my friends are
not interested in Diaspora).

I'm not open to argue about my reasons (I have other things to care about)
but I can share you more details if you ask me.

On Friday, January 23, 2015, goob <notifications@github.com> wrote:

> @maliktunga <https://github.com/maliktunga> It's a shame that you don't
> want to continue contributing. Is there any particular reason for that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/5579#issuecomment-71246650>.
>

RichKavanagh commented:
Are you @GP on Loomio?

maliktunga commented:
Yes.

On Saturday, January 24, 2015, Rich <notifications@github.com> wrote:

> Are you @GP <https://github.com/GP> on Loomio?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/5579#issuecomment-71302565>.
>

goobertron commented:
Thanks for your explanation. I just thought, they way you dropped this so fast, you might have taken offence at something someone said, so I wanted to check you hadn't decided to stop contributing based on a misunderstanding. Good luck in whatever you do.

maliktunga commented:
Thank you.

On Saturday, January 24, 2015, goob <notifications@github.com> wrote:

> Thanks for your explanation. I just thought, they way you dropped this so
> fast, you might have taken offence at something someone said, so I wanted
> to check you hadn't decided to stop contributing based on a
> misunderstanding. Good luck in whatever you do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/5579#issuecomment-71309678>.
>

goobertron commented:
I notice this has been given an 'orphan' label. To save anyone else starting on this, I'm completing it.

svbergerem commented:
#5588 has been merged so I guess we can close this. @maliktunga Thank you for your work.
